main.go

```diff
@@ -116,6 +120,12 @@ func main() {
 	flag.StringVar(&c.saName, "sa-name", c.saName, "ServiceAccount name for cleanup Pod/DaemonSet/Job")
 	flag.StringVar(&c.roleName, "role-name", c.roleName, "Role name for cleanup Pod/DaemonSet/Job")
 	flag.StringVar(&c.roleBindingName, "role-binding-name", c.roleBindingName, "RoleBinding name for cleanup Pod/DaemonSet/Job")
+	flag.StringVar(&c.clusterRoleName, "cluster-role-name", c.roleName, "ClusterRole name for cleanup Pod/DaemonSet/Job. If set, role-name will be ignored.")
+	flag.StringVar(&c.clusterRoleBindingName, "cluster-role-binding-name", c.roleBindingName, "ClusterRoleBinding name for cleanup Pod/DaemonSet/Job. If set, role-binding-name will be ignored.")
+
+	if c.clusterRoleName == "" && c.clusterRoleBindingName != "" || c.clusterRoleName != "" && c.clusterRoleBindingName == "" {
+		panic("cluster-role-name and cluster-role-binding-name must be set together")
```

Comment:
Couldn't we just exit `1` and output a message vs panicking and returning the stack track that comes with a panic? Perhaps this is a team norm that I'm just not aware of

Reply:
more or less im just following the patterns that already exist in the repo. i think its a valid call out that maybe we should refactor to do what youre suggesting though. id just rather we do that in a separate PR.